deploy/probers/base.py

```diff
@@ -45,6 +45,11 @@ def _login(self):
         )
         r.raise_for_status()
 
+        # Since logging into Studio with wrong username and password also returns 200
+        # status code, check the response url to see whether we have logged in or not.
+        if r.url == url:
```

Comment:
Ah i'm assuming this means we got redirected back to the same login page?

Reply:
yes otherwise it will go to `/channels` page.